owlbot.py

```diff
@@ -67,4 +69,6 @@
 # Run blacken session
 # ----------------------------------------------------------------------------
 
-s.shell.run(["nox", "-s", "blacken"], hide_output=False)
+# run blacken session for all directories which have a noxfile
+for noxfile in Path(".").glob("**/noxfile.py"):
+    s.shell.run(["nox", "-s", "blacken"], cwd=noxfile.parent, hide_output=False)
```

Comment:
This will break if the noxfile doesn't have a `blacken` sesssion, but the only case I can think of is google-auth-library-python's system test noxfile (and they have a separate `owlbot.py` anyway).